framework/lib/components/createable-select-dropdown/custom-components.tsx

```diff
@@ -0,0 +1,21 @@
+import React from 'react'
+import { chakraComponents } from 'chakra-react-select'
+import { PlusSolid } from '@northlight/icons'
+import { Box, Flex, Icon } from '@chakra-ui/react'
+
+export const customComponents = {
+  Option: ({ children, ...props }: any) => (
+    <chakraComponents.Option { ...props }>
+      <>
+        { props.data.value === 'custom-option' ? (
+          <Flex mr={ 2 } width={ 1.5 } justifyContent="center" alignItems="center">
```

Comment:
And you prob want to increase the mr from 2 to 3

Reply:
done! also added a small mb to horizontally align the + sign